libs/sdk-backend-mockingbird/src/recordedBackend/execution.ts

```diff
@@ -110,6 +112,9 @@ function recordedPreparedExecution(
         withSorting(...items: ISortItem[]): IPreparedExecution {
             return executionFactory.forDefinition(defWithSorting(definition, items));
         },
+        withPostProcessing(postProcessing: IPostProcessing): IPreparedExecution {
```

Comment:
You also need to change the `execute` function, because now that the postProcessing is not a part of the recording, we need to take it from the original execution definition.

Reply:
I'm sorry but could you please explain this comment a little bit, what should be changed in the execute function. Thank you.